index.html

```diff
@@ -1,304 +1,327 @@
-                </p>
-            </div>
-            <form action="https://formspree.io/html" id="form" class="form" method="post">
-                <ul class="inner-list">
+      <section class="sectorcontact">
+        <div class="contact_sub">
+            <div class="cont_act">
+                <div class="contact_items">
+                    <h2 class="contactsheader" id="contact">Contact me</h2>
+                    <p class="contact_paragraph">If you have an application you are interested in developing, a
+                    feature that you need built or a project that needs coding. I’d
+                    love to help with it</p>
+                </div>
+                <div class="form">
+                  <form action="https://formspree.io/f/noqsnoqs" method="POST" id="form" class="form">
+                  <ul class="form-list">
                     <li>
-                        <input type="text" class="form-control" placeholder="Harrison" name="name" id="name" maxlength="30" required />
+                      <input
+                        type="text"
+                        class="name"
+                        placeholder="Harrison"
+                        name="name"
+                        id="name"
+                        maxlength="30"
+                        required/>
                     </li>
                     <li>
-                        <input type="email" class="form-control" placeholder="harry@gmail.com" name="email" id="email" pattern="email" required />
+                      <input
+                        type="email"
+                        class="email"
+                        placeholder="harry@gmail.com"
+                        name="email"
+                        id="email"                        
+                        required
+                      />
                     </li>
                     <li>
-                        <textarea name="text" class="form-control" id="text" placeholder="Write your message here" maxlength="500" required></textarea>
-                    </li>
-                    <li class="flex-item6" id="submit-btn">
-                        <button type="submit">Get In Touch</button>
-                    </li>
-                    <li class="shape"><img src="./images/Shape.png" alt="shape" /></li>
-                </ul>
-            </form>
-        </section>
+                      <textarea
+                        name="text"
+                        class="form_message"
+                        id="text"
+                        placeholder="Write your message here"
+                        maxlength="500"
+                        required
+                      ></textarea>
+                    </li>                   
+                  </ul>
+                  <div class="get_container"><input class="see-button" type="button" value="Get in touch" id="sbm">
```

Comment:
- [ ] The submission to Formspree isn't working because the value for the `type` attribute is incorrect. Please use the `button` for the submission button `type` attribute. Hint: `type="submit`